src/templates/common/benefitsHubLinks/index.tsx

```diff
@@ -1,41 +0,0 @@
-import { recordEvent } from '@/lib/analytics/recordEvent'
```

Comment:
Removed all BenefitsHubLink things as it became a common component (RelatedLink(s))